anchor/programs/depo/src/instructions/add_recipient.rs

```diff
@@ -0,0 +1,51 @@
+use anchor_lang::prelude::*;
+use crate::states::{Escrow, Recipient};
+use crate::errors::EscrowErrors;
+/// Adds a recipient to the escrow
+///
+/// # Arguments
+/// * `ctx` - The context containing the account and signer
+/// * `_escrow_id` - The unique identifier (UUID) for the escrow
+///
+/// # Returns
+/// * `Result<()>` - Result indicating success or failure
+pub fn add_recipient(
+    ctx: Context<AddRecipientCtx>,
+    _escrow_id: [u8; 16]
+) -> Result<()> {
+    let recipient = &mut ctx.accounts.recipient;
+    recipient.escrow = ctx.accounts.escrow.key();
+    recipient.wallet = ctx.accounts.signer.key();
+    recipient.amount = 0;
+    recipient.has_withdrawn = false;
+
+    let escrow = &mut ctx.accounts.escrow;
+    escrow.recipients_count += 1;    
+
+    Ok(())
+}
+
+#[derive(Accounts)]
+#[instruction(escrow_id: [u8; 16])]
+pub struct AddRecipientCtx<'info> {
+    #[account(
+      mut,
+      seeds = [b"escrow", escrow_id.as_ref()],
+      bump
+    )]
```

Comment:
Should we add has_one = initializer to be sure only the person who created the escrow can modify it ?